projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/answers/AnswerSummary.java

```diff
@@ -0,0 +1,84 @@
+package org.batfish.datamodel.answers;
+
+import com.google.common.base.Strings;
+
+public class AnswerSummary {
```

Comment:
I think this needs to be JSON Serializable -- given that `AnswerSummary` is now a `@JsonProperty` of `AnswerElement` entity.

Reply:
If you don't do @JsonProperty, Jackson will use field names based on getter and setter names. I started using @JsonProperty because I was worried about people changing getter and setter names, but it does create a lot of clutter and extra LoC. I now think a better way to deal is to have serialization and deserialization unit tests, which I added for this class. 

If you think @JsonProperty buys us something, I can add those annotations.